frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetails.tsx

```diff
@@ -221,7 +234,83 @@ export function PolicyTemplateDetails() {
     }
 
     return cols
-  }, [t, name, kind, apiGroup, template, clusterName, vapb, apiVersion])
+  }, [t, name, namespace, kind, apiGroup, template, clusterName, vapb, apiVersion])
+
+  const violationColumn = useMemo(() => {
```

Comment:
Did we discuss this? I thought we decided to add to beside header

Reply:
This is to show which objects are in violation in the "Related resources" table since Sho wanted the passing results to be shown in that table also. Before it was unnecessary to have this column since all entries were of violations.